v4/source/introduction.yaml

```diff
@@ -300,22 +300,27 @@ tags:
 
   - name: APIv3 Deprecation
     description: |
-      With the release of Mattermost server 4.0, API version 4 is stable and we strongly recommend using it over API version 3. The deprecation of API version 3 is planned for January 16th, 2018, which is six months after the stable release of v4.
-
-      The major changes from v3 to v4 are:
-
-      - Endpoint URLs only require team IDs when necessary, for example getting a channel by ID does not require a team ID in v4
-      - Collection endpoints now generally return lists and include paging as part of the query string
-      - User ID is now included in most user endpoints to allow for admins to use these endpoints to modify other users
-
-      All new integrations should use v4 and existing ones should begin migration over to version 4. Some suggested steps to help migrate your server and integrations are:
-
-      1. Set your server's log level to `DEBUG` and look for any requests hitting `/api/v3/` endpoints. The Mattermost webapp, desktop app, and classic mobile apps are solely using API version 4 and the new mobile apps will be completely moved over in their next release. That means that the majority of the logs you see will be from integrations that need to be migrated to v4.
-      2. For in-house or self-built integrations, begin updating them to use v4 with the help of this documentation. Most v3 endpoints have direct counterparts in v4 and should be easily migrated.
-      3. For integrations built by a third-party, visit their homepage (on GitHub, GitLab, etc.) and check if they already have a version that uses the Mattermost v4 API. If they do not, consider opening an issue asking them if support is planned.
-      4. Once you have migrated all your integrations to use v4, take another look at the server logs to make sure you don't see many v3 endpoints in use. Then, in your `config.json`, set the setting `EnableAPIv3` to `false`. In the system console this setting is under `General` -> `Configuration` -> `Allow use of API v3 endpoints`. This will disable v3 and output an error to your server logs any time one is attempted to be used. Set your server's log level back to `ERROR` and use the error logs to help track down any last uses of v3.
-
-      If you have any questions about the deprecation or about migrating from v3 to v4, feel free to [join our daily build server at pre-release.mattermost.com](https://pre-release.mattermost.com/signup_user_complete/?id=f1924a8db44ff3bb41c96424cdc20676) and ask questions in the [APIv4 channel](https://pre-release.mattermost.com/core/channels/apiv4).
+      Since Mattermost 4.0, API v4 is the preferred version to be used. API v3 are unsupported since Mattermost 4.6, released on January 16, 2018. They will be removed in Mattermost 5.0.
+      
+      Mattermost APIs are primarily used for integrations, including webhooks and custom slash commands. We advice that any new integrations use API v4 endpoints.
```

Comment:
I don't think the `including webhooks and custom slash commands` bit makes sense. Any integrations using solely slash commands and webhooks are API version independent and don't need to worry about API version changes

And the Mattermost API isn't primarily used for integrations. We use it primarily for powering our clients (and so does Matterhorn for example)

Maybe just remove the first sentence of this line